src/Xrefcheck/Scanners/Markdown.hs

```diff
@@ -15,15 +15,17 @@ module Xrefcheck.Scanners.Markdown
   , parseFileInfo
   ) where
 
+import Universum
+
 import CMarkGFM (Node (..), NodeType (..), PosInfo (..), commonmarkToNode)
 import Control.Lens ((%=))
 import Control.Monad.Trans.Except (Except, runExcept, throwE)
 import Data.Aeson.TH (deriveFromJSON)
-import qualified Data.ByteString.Lazy as BSL
+import Data.ByteString.Lazy qualified as BSL
 import Data.Char (isSpace)
 import Data.Default (Default (..))
-import qualified Data.Text as T
-import qualified Data.Text.Lazy as LT
```

Comment:
This is not accepted yet) but I do not mind
Can you 'vote' here?
https://github.com/serokell/style/pull/11

Reply:
I use `ImportQualifiedPost` that mitigates the problem anyway.